### PR TITLE
Add a post about ember controllers

### PR DESCRIPTION
**Pull request by mitchlloyd**
Hey you guys, please proof-read this and check out the examples.

cc @rockwood @mguterl @superchris

### CODE REVIEW

File: posts/2013-07-01-ember-controller-concepts.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+which is reflected in the template to the item get the `is-selected` class.
+
+But oh no! If we click another item it also gets selected without unselected the
+other one. [Try it out](http://jsbin.com/ikusok/3).
+
+We need a way to unselect names. Let's introduce a controller to manage the
+list.
+
+```coffee
+App.IndexController = Ember.ArrayController.extend
```
Comment (mguterl) [inline posts/2013-07-01-ember-controller-concepts.md:93]: @mitchlloyd - this is just a reminder about our discussion on how moving from NameController to IndexController and name being specified is a big jump and you may want to simplify this a bit.

**Comment by mitchlloyd** [comment]
@joelturnbull this is ready to be merged and published whenever fits best into the blog schedule.

**Comment by cdmwebs** [comment]
:metal:

File: posts/2013-07-01-ember-controller-concepts.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+App.IndexRoute = Ember.Route.extend
+  model: ->
+    [
+      {firstName: 'Gut', lastName: 'Bomb'}
+      {firstName: 'Rock', lastName: 'Tastic'}
+      {firstName: 'The', lastName: 'Fish'}
+    ]
+```
+
+```handlebars
```
Comment (cdmwebs) [inline posts/2013-07-01-ember-controller-concepts.md:47]: handlebars

**Comment by joelturnbull** [comment]
Good stuff, totally clear to me. The jsbin stuff is perfect.

The title seems vague. How about "Ember Controllers as Model Decorators", or "Ember: How to Decorate a Model with a Controller", or something more specific.

**Comment by mitchlloyd** [comment]
Brace yourself for the lamest follow up ever... How about "Intermediate Ember Controller Concepts"?

That title is appealing to me as someone who is trying to learn the ins and outs of Ember. The titles about Ember controllers as model decorators actually would turn me off because I know that controllers decorate models and I wouldn't expect much substance in a post with that title.  "Intermediate Ember Controller Concepts" makes me want to click on it because I feel like I'll probably learn something new past the usual "introduction to Ember" tutorials.

What do you think?

**Comment by joelturnbull** [comment]
Ok :)

**Comment by rockwood** [comment]
+1

On Tuesday, July 2, 2013, Joel Turnbull wrote:

> Ok :)
> 
> On Tuesday, July 2, 2013, Mitch Lloyd wrote:
> 
> > Brace yourself for the lamest follow up ever... How about "Intermediate
> > Ember Controller Concepts"?
> > 
> > That title is appealing to me as someone who is trying to learn the ins
> > and outs of Ember. The titles about Ember controllers as model
> > decorators
> > actually would turn me off because I know that controllers decorate
> > models
> > and I wouldn't expect much substance in a post with that title.
> > "Intermediate Ember Controller Concepts" makes me want to click on it
> > because I feel like I'll probably learn something new past the usual
> > "introduction to Ember" tutorials.
> > 
> > What do you think?
> > 
> > —
> > Reply to this email directly or view it on GitHub<
> > https://github.com/gaslight/posts/pull/23#issuecomment-20392639>
> > .
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/gaslight/posts/pull/23#issuecomment-20392739
> .

## 

Kevin Rockwood | www.rockwood.me

**Pull request merged**